src/Components/DraftFeed/Components/DraftablePlayers/DraftablePlayers.js

```diff
@@ -0,0 +1,31 @@
+import './DraftablePlayers.css';
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+export var players = [];
+
+export function DraftablePlayers({ name, position, FFP }) {
+  const [, setPlayer] = useState([]);
+  function addPlayerToTeam(e) {
+    const item = e.target.value;
+    players.push(name);
+    document.getElementsByClassName('updateDraft')[0].click();
+    setPlayer((team) => [...team, item]);
+  }
+
+  return (
+    <div className="draftablePlayers">
+      <NavLink
+        to="/Draft"
+        className="updateDraft"
+        activeClassName="updateDraft"
+      ></NavLink>
+      <h4 className="draftablePlayers__text">
+        <strong>Name: </strong> {name}
+        <strong>Pos: </strong> {position}
+        <strong>FFP: </strong>
+        {FFP}
+        <button onClick={addPlayerToTeam}>+</button>
+      </h4>
```

Comment:
we probably should move the `<button>` outside of the `<h4>` since the plus symbol isn't really part of a title

Reply:
Resolved